app/controllers/avo/associations_controller.rb

```diff
@@ -200,7 +200,7 @@ def through_reflection?
     end
 
     def additional_params
-      @additional_params ||= params[:fields].permit(@attach_fields&.map(&:id))
+      @additional_params ||= params[:fields].slice(*@attach_fields&.map(&:id))
```

Comment:
Applying `permit` only on the `attach_fields` would raise the error of the unpermitted parameters for the remaining parameters.

`permit` is unnecessary here because `fill_record` enures that only params that match `attach_fields` ids will be sent to the record.